packages/core/bootstrap/src/lib/cache-warmer/epics.ts

```diff
@@ -174,18 +174,44 @@ export const warmupRequestHandler: Epic<AnyAction, AnyAction, any> = (action$, s
     filter(warmupRequested.match),
     // fetch our required state to make a request to warm up an adapter
     withLatestFrom(state$),
-    map(([action, state]) => ({
-      requestData: state.cacheWarmer.subscriptions[action.payload.key],
-      key: action.payload.key,
-    })),
+    map(([action, state]) => {
+      const requestData = state.cacheWarmer.subscriptions[action.payload.key]
+      let batchablePropertyPath
+      if (requestData && requestData.childLastSeenById && Object.keys(requestData.childLastSeenById).length > 0) {
```

Comment:
```suggestion
      if (requestData?.childLastSeenById && Object.keys(requestData.childLastSeenById).length > 0) {
```
optional chaining makes this a bit more concise.

also could use lodash.isEmpty as a more robust way to check for an empty object